IngestionGrain/GrianFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Orleans;
```

Comment:
This should be refactored into its own project in the root of the solution. Project name: Orleans, Base namespace: CommunAxiom.Commons.Orleans
This proxy will be used anywhere  a grain refers to another in both Silo and the Orchestrator